swagger/swagger.json

```diff
@@ -543,77 +548,37 @@
       "description" : "A structure representing the payment card"
     },
     "CardPayment" : {
-      "type" : "object",
-      "properties" : {
-        "amount" : {
-          "type" : "integer",
-          "format" : "int64",
-          "example" : 1200
-        },
-        "state" : {
-          "$ref" : "#/definitions/PaymentState"
-        },
-        "description" : {
-          "type" : "string",
-          "example" : "Your Service Description"
-        },
-        "reference" : {
-          "type" : "string",
-          "example" : "your-reference"
-        },
-        "email" : {
-          "type" : "string",
-          "example" : "your email"
-        },
-        "payment_id" : {
-          "type" : "string",
-          "example" : "hu20sqlact5260q2nanm0q8u93",
-          "readOnly" : true
-        },
-        "payment_provider" : {
-          "type" : "string",
-          "example" : "worldpay",
-          "readOnly" : true
-        },
-        "return_url" : {
-          "type" : "string",
-          "example" : "http://your.service.domain/your-reference",
-          "readOnly" : true
-        },
-        "created_date" : {
-          "type" : "string",
-          "example" : "2016-01-21T17:15:00Z",
-          "readOnly" : true
-        },
-        "refund_summary" : {
-          "readOnly" : true,
-          "$ref" : "#/definitions/RefundSummary"
-        },
-        "settlement_summary" : {
-          "readOnly" : true,
-          "$ref" : "#/definitions/SettlementSummary"
-        },
-        "card_details" : {
-          "readOnly" : true,
-          "$ref" : "#/definitions/CardDetails"
-        },
-        "delayed_capture" : {
-          "type" : "boolean",
-          "example" : false,
-          "description" : "delayed capture flag",
-          "readOnly" : true
-        },
-        "_links" : {
-          "readOnly" : true,
-          "$ref" : "#/definitions/PaymentLinksForSearch"
-        },
-        "card_brand" : {
-          "type" : "string",
-          "example" : "Visa",
-          "description" : "Card Brand",
-          "readOnly" : true
+      "allOf" : [ {
```

Comment:
I guess these changes were not synched to `swagger.json` that is why they are not related to the PR